src/components/settingpage/leftcomponent/dialogbox.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import classnames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+interface IProps {
+  changestate: (display: boolean) => void;
+}
+
+const DialogBox: React.FC<IProps> = ({ changestate }) => {
+  const { t } = useTranslation();
+
+  const title = classnames('setting-left--dialogtitle');
+  const passwordinput = classnames('setting-left--passwordInput');
+
+  const [oldPassword, changeoldPassword] = useState<string>('');
+  const [newPassword, changenewPassword] = useState<string>('');
+  const [confirmPassword, changeconfirmPassword] = useState<string>('');
+
+  const changeRequest = async () => {
+    const password = JSON.stringify({
+      original_passwd: oldPassword,
+      new_passwd: newPassword,
+    });
+    // get bearer token
+    const user = JSON.parse(
+      localStorage.getItem('BEMS_USER') ||
+        sessionStorage.getItem('BEMS_USER') ||
+        '{}',
+    );
+    // GET to User Info API
+    const response = await fetch(
+      `${process.env.REACT_APP_BACKEND_ENDPOINT}/user`,
+      {
+        method: 'PUT',
+        mode: 'cors',
+        headers: new Headers({
+          Authorization: `Bearer ${user.bearer}`,
+          'Content-Type': 'application/json',
+        }),
+        body: password,
+      },
+    );
+    if (response.status === 200) {
+      // fetch success
+      window.alert(t('settingpage.successChange'));
```

Comment:
If you need to use alert, you need to disable eslint rule for this line.
You can do this by hovering the warning and click `Quick fix` then choose `Disable ... for this line`.
Same comments for alert below.

Reply:
done